json-sql

```diff
@@ -53,8 +66,10 @@ class DB:
 
     def read_table(self, table_name):
         columns = self._column_names(table_name)
-
-        data = self.execute_query("select * from {};".format(table_name))
+        if self.__class__.__name__ == 'ORACLEADB':
```

Comment:
Can we change this to override the read_table method in the ORACLEDB class instead of putting an if statement here?